README.rst

```diff
@@ -24,7 +21,26 @@ e.g., to get driver information on the ``eth0`` interface::
     driver: cdc_ether
     bus-info: usb-0000:00:14.0-4.1.3
 
-Further usage information may be found in the ``pethtool(8)`` manpage.
+Analogically, ``pifconfig`` mimics ``ifconfig`` in usage.  It may be
+used to view information on an interface::
+
+    $ pifconfig lo
+    lo        
+          inet addr:127.0.0.1   Mask:255.0.0.0
+	      inet6 addr: ::1/128 Scope: host
+	      UP LOOPBACK RUNNING
+
+
+Further usage information may be found in the respective manpages for
+`pethtool <https://github.com/fedora-python/python-ethtool/blob/master/man/pethtool.8.asciidoc>`_
+and
+`pifconfig <https://github.com/fedora-python/python-ethtool/blob/master/man/pifconfig.8.asciidoc>`_.
+
```

Comment:
Since we advertise this as "Python bindings for the ethtool kernel interface", it would be great to show an usage from Python itself. However, that might be too hard for now, so i'd suggest add something like: to see an usage from Python, peak tot he source codes of pethtool and pifconfig (with links).